src/aat/java/uk/gov/hmcts/ccd/datastore/befta/DataLoader.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.ccd.datastore.befta;
+
+import uk.gov.hmcts.befta.dse.ccd.DataLoaderToDefinitionStore;
+
+import java.security.InvalidParameterException;
+
+public class DataLoader extends DataLoaderToDefinitionStore {
+
+    public DataLoader(String dataSetupEnvironment) {
+        super(dataSetupEnvironment);
+    }
+
+    public static void main(String[] args) {
+        if (args.length == 0) {
+            throw new InvalidParameterException();
```

Comment:
let's use another Exception. `java.security.InvalidParameterException` is not relevant.

Reply:
changed it with IllegalArgumentException